nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core.Test/Utilities/MSBuildHelperTests.cs

```diff
@@ -187,6 +187,29 @@ public static IEnumerable<object[]> GetTopLevelPackageDependenyInfosTestData()
             }
         };
 
+        // version is a child-node of the package reference
+        yield return new object[]
```

Comment:
This already worked but I was already adding tests for this scenario so why not here too.